lib/database_cleaner/active_record/truncation.rb

```diff
@@ -14,6 +14,7 @@ def initialize(opts={})
 
         @reset_ids = opts[:reset_ids]
         @pre_count = opts[:pre_count]
+        @truncate_option = opts[:cascade] || :restrict
```

Comment:
I'm pretty sure this should be `opts[:truncate_option]` rather than `opts[:cascade]` :)